pkg/apis/ceph.rook.io/v1/validate_resources_test.go

```diff
@@ -0,0 +1,127 @@
+package v1
+
+import (
+	"github.com/ghodss/yaml"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestCephClusterValidateCreate(t *testing.T) {
+	cephClusterYaml := getCephClusterCRD()
+	c := &CephCluster{}
+	err := yaml.Unmarshal(cephClusterYaml, c)
+	assert.Nil(t, err)
+	err = c.ValidateCreate()
+	assert.Nil(t, err)
```

Comment:
Instead of `assert.Nil()`, it's preferred to use `assert.NoError()`
```suggestion
	assert.NoError(t, err)
```

Reply:
fixed